pl_bolts/datasets/ssl_amdim_datasets.py

```diff
@@ -3,12 +3,14 @@
 
 import numpy as np
 
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
 from pl_bolts.utils.warnings import warn_missing_pkg
 
-try:
+if _TORCHVISION_AVAILABLE:
     from torchvision.datasets import CIFAR10
-except ModuleNotFoundError:
-    warn_missing_pkg('torchvision')  # pragma: no-cover
+else:  # pragma: no cover
+    warn_missing_pkg('torchvision')
+    CIFAR10 = object
```

Comment:
```suggestion
```

Reply:
We actually need this line to ensure to import the subclass successfully even without `torchvision` or other optional packages.

Reply:
I found it also and already revered that suggestion, sorry :]